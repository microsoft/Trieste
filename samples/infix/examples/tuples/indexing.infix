tpl1 = (1,2,3);
print "1" tpl1.0;
print "2" (42, tpl1, 42).1.(1); // () to avoid writing a float
print "3" tpl1.(tpl1.1);