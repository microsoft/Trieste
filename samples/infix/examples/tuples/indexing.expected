//! --enable-tuples --tuples-require-parens calculate
//! --enable-tuples --tuples-require-parens --use-parser-tuples calculate
//! --enable-tuples calculate
1 <- 1
2 <- 2
3 <- 3
